cmd/build-uki.go

```diff
@@ -93,7 +93,8 @@ func NewBuildUKICmd() *cobra.Command {
 				return fmt.Errorf("keys directory does not exist: %s", keysDir)
 			}
 			// Check if the keys directory contains the required files
-			requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
+			//requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
+			requiredFiles := []string{"db.auth", "KEK.auth", "PK.auth"}
```

Comment:
I'm not 100% sure about this. With `auth` files around, the `der` files don't seem to be needed anywhere.

Reply:
I did the work to append the custom certs also in the der files but when I manually enroll the certs in my Asus PN64 pointing to the der files, I only get one cert imported. I guess it only takes the first one in the file (?). In any case, auto-enrollment with systemd-boot still works, either because it uses the "auth" files or because it parses the der files differently.

It's probably ok to keep this PR in its current state.